doc/tutorials/transformer/masked_modelling.ipynb

```diff
@@ -0,0 +1,235 @@
+{
```

Comment:
<p>However, the main purpose of having a model deployed in the database is to get a quick response for a <s>batch</s> whole table.</p><p> </p><p>localization and parallelization -&gt; localization has usually a different meaning (<a href="https://en.wikipedia.org/wiki/Language_localisation" target="_blank">https://en.wikipedia.org/wiki/Language_localisation</a>), maybe a better word is data locality</p><p> </p><pre class="ql-syntax" spellcheck="false">SELECT TE_FILLING_MASK_UDF(..., MY_TEXT_COLUMN, 1) FROM MY_TEXT_TABLE GROUP BY IPROC(), mod(rowid, instance_per_node)); 
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>